packages/build/src/plugins_core/deploy/buildbot_client.js

```diff
@@ -84,7 +84,18 @@ const getDeployDir = function ({ buildDir, repositoryRoot, constants: { PUBLISH_
 
 // We distinguish between user errors and system errors during deploys
 const handleDeployError = function (error, errorType) {
```

Comment:
would you mind adding a jsdoc type as you figuerd the structure out?

Reply:
I just know that the structure is an object but I don't know what properties it can have.